src/coreclr/tools/aot/ILCompiler/ILCompilerRootCommand.cs

```diff
@@ -160,6 +160,8 @@ internal sealed class ILCompilerRootCommand : RootCommand
             new(new[] { "--singlemethodgenericarg" }, "Single method compilation: generic arguments to the method");
         public Option<string> MakeReproPath { get; } =
             new(new[] { "--make-repro-path" }, "Path where to place a repro package");
+        public Option<string[]> RootedAssembliesExports { get; } =
+            new(new[] { "--rootedassembliesexports" }, Array.Empty<string>, "Root exported methods from assemblies");
```

Comment:
nit: perhaps singular name would be suitable? Just like `<IlcArg Include=... />`, `--initassembly`, `--appcontextswitch`, `--feature` etc. all have zero or more arity.

Reply:
I agree the naming is not ideal. I will update the switch based on: https://github.com/dotnet/runtime/pull/83396#discussion_r1136171171